spec/factories/service_providers.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :service_provider do
     sequence(:friendly_name) { |n| "test-service_provider-#{n}" }
-    sequence(:issuer) { |n| "test-service_provider-#{n}" }
+    sequence(:issuer) { |n| "urn:gov:gsa:SAML:2.0.profiles:sp:sso:DEPT:APP-#{n}" }
```

Comment:
the issuer is our primary key in the IDP, I really don't think we should force SPs to change since there would be no continuity? That would cause use to generate new UUIDs for existing users for the SPs?

This is the dashboard, so it only affects lower envs, maybe that's OK? Happy to chat more about this in the orignal issue if needed